krun/vm_defs.py

```diff
@@ -480,12 +579,105 @@ def __init__(self, vm_path, env=None):
         lib_dir = os.path.dirname(vm_path)
         self.add_env_change(EnvChangeAppend("LD_LIBRARY_PATH", lib_dir))
 
+    def parse_instrumentation_stderr_file(self, file_handle):
+        """PyPy instrumentation data collected from the PYPYLOG.
+
+        We collect:
+          * The time spent in compilation-related tasks.
+          * The time spent in GC-related tasks.
+
+        Note that the times are not in wall-clock time. Consider the units
+        arbitrary.
```

Comment:
The times can be used relatively even if we do not know the units. E.g.

```
In iteration X the VM spent twice as long compiling than in iteration Y. This is reflected by a larger spike in the iteration time for X.
```

Regardless, I'm going to read the PyPy code to try to find out what the time is in terms of. At that point, we can revise this comment.